account-kit/react/src/components/auth/card/add-passkey.tsx

```diff
@@ -48,7 +49,7 @@ export const AddPasskey = ({ config }: { config: AuthCardProps }) => {
         {BENEFITS.map(({ title, icon: Icon, description }) => (
           <div key={title} className="flex gap-2">
             <div className="h-5 w-5 flex items-center justify-center">
-              <Icon illustrationStyle={config.illustrationStyle ?? "flat"} />
+              <Icon illustrationStyle={illustrationStyle ?? "flat"} />
```

Comment:
With the hook, we could move this prop directly into the Icon components which would be kinda nice